phage.nf

```diff
@@ -879,7 +879,7 @@ def helpMSG() {
                         [default: ${params.databases}]
                         ${c_dim}WtP downloads DBs if not present at this path${c_reset}
 
-    --workdir           defines the path where nextflow writes temporary files 
+    -workdir           defines the path where nextflow writes temporary files 
```

Comment:
no its `--workdir` as this is our defined `params.workdir` not the built-in `-work-dir`